### ISSUE TITLE
Could you please publish a package?

### ISSUE THREAD

**Issue by mrpmorris** [davidfowl/Roslyn.Reflection#5]
Hi David

Thanks for your library!

Could you please publish a NuGet package? If like to use your library in a code generator, and would like to depend on your package instead of my own temporary one.

Thanks!

**Comment by davidfowl** [comment]
Yes, I've been meaning to do that. I did publish it to a CI feed though.

**Comment by mrpmorris** [comment]
I'd like to publish a package onto NuGet that depends on it. So NuGet would be best option, if you wouldn't mind?

**Comment by davidfowl** [comment]
<img width="893" alt="image" src="https://user-images.githubusercontent.com/95136/204927952-cb59d50a-b3c5-4e58-9ce8-b89785e88ee6.png">

Turns out somebody pushed before me?

**Comment by davidfowl** [comment]
Or maybe the Roslyn prefix is taken?

**Comment by davidfowl** [comment]
Yep, I need to rename it

**Comment by marcusbooyah** [comment]
Is there an update on this? I'd also love to see this on Nuget!
